app/spaces/[spaceid]/trustful/share/components/QRCodeGiveBadge.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import { useQRCode } from 'next-qrcode';
+import { useParams } from 'next/navigation';
+import { useAccount } from 'wagmi';
+
+export const QRCodeGiveBadge = () => {
+  const { Canvas } = useQRCode();
+  const { address } = useAccount();
+  const params = useParams();
+
+  //TODO: Check if the link works in production development and it's redirecting to zuzalu page correctly.
+  const isDev = process.env.NEXT_PUBLIC_ENV;
+  const prefixToGiveBadge = `${isDev ? `http://localhost:3000/spaces/${params.spaceid}/trustful` : `http://localhost:3000/spaces/${params.spaceid}/trustful`}`;
```

Comment:
This is not correct as both URLs redirect to localhost.

Reply:
Yes. It's a TODO [38](https://github.com/blockful-io/frontend-zucity/issues/38)